controllers/servicefence_controller.go

```diff
@@ -560,6 +706,13 @@ func isValidHost(h string) bool {
 	return true
 }
 
+func isValidNs(ns string, env bootstrap.Environment) bool {
```

Comment:
`isValid` does not match the purpose of this function.
also we should keep consistency of the logic here and the code where we add these namespaces below.

for example:

```go
type ServicefenceReconciler struct {
  defaultAddNamespaces []string  // or map[string]struct{} if u like
}

// use env.Config to init defaultAddNamespaces field

func (r *ServicefenceReconciler) isDefaultAddNs(ns string) bool {
  for _, defaultNs := range r.defaultAddNamespaces {
    if defaultNs == ns {
      return true
    }
  }
  return false
}

for _, ns := range r.defaultAddNamespaces {
  hosts = append(hosts, ns+"/*")
}
```